src/stp/skills/GoToPos.cpp

```diff
@@ -31,21 +25,19 @@ Status GoToPos::onUpdate(const StpInfo &info) noexcept {
     Vector2 targetVelocity = robotCommand.vel.stretchToLength(targetVelocityLength);
 
     // Set velocity and angle commands
-    command.mutable_vel()->set_x(targetVelocity.x);
-    command.mutable_vel()->set_y(targetVelocity.y);
+    command.mutable_vel()->set_x(static_cast<float>(targetVelocity.x));
+    command.mutable_vel()->set_y(static_cast<float>(targetVelocity.y));
```

Comment:
We cast here because targetVelocity.x is a double and we need a float, right? Otherwise we get nice yellow warnings and this avoids those?

Reply:
Yep, exactly